
<launch>

	<arg name="mav_name" default="quad"/>
	<arg name="id" default="2"/>
	<arg name="quad_ns"  default="$(arg mav_name)$(arg id)"/>
	<arg name="fixed_height" default="1.5"/>
	<arg name="max_vel" default="0.5"/>
  <arg name="auto_takeoff" default="false"/>
	
	<group ns="$(arg mav_name)$(arg id)">
		<node pkg="mslquad" type="default_controller" name="default_controller" output="screen">
      <param name="controller_type" value="se3coop"/>
			<param name="quad_ns" value="$(arg quad_ns)"/>
			<param name="fixed_height" value="$(arg fixed_height)" />
			<param name="max_vel" value="$(arg max_vel)" />
			<param name="control_freq" value="250"/>
			<param name="slow_freq" value="10"/>
      <param name="auto_takeoff" value="$(arg auto_takeoff)"/>
      <param name="KP" value="6.0" />
      <param name="KV" value="4.0" />
      <param name="KR" value="2.0" />
      <param name="KW" value="0.1" />
      <param name="M" value="2.635" />
      <param name="quadFrame" value="mslquad"/>
			<param name="quad_addr" value="0"/>    
			<param name="quad_count" value="2"/>  
			<param name="arm_length" value="0.35"/>    
		</node>

	</group>
</launch>